package.json

```diff
@@ -30,21 +30,21 @@
     "accepts": "~1.2.7",
     "content-disposition": "0.5.0",
     "content-type": "~1.0.1",
-    "cookie": "0.1.2",
+    "cookie": "0.1.3",
     "cookie-signature": "1.0.6",
     "debug": "~2.2.0",
     "depd": "~1.0.1",
-    "escape-html": "1.0.1",
+    "escape-html": "1.0.2",
     "etag": "~1.6.0",
     "finalhandler": "0.3.6",
-    "fresh": "0.2.4",
+    "fresh": "0.3.0",
     "merge-descriptors": "1.0.0",
     "methods": "~1.1.1",
-    "on-finished": "~2.2.1",
+    "on-finished": "~2.3.0",
     "parseurl": "~1.3.0",
-    "path-to-regexp": "0.1.3",
+    "path-to-regexp": "0.1.5",
     "proxy-addr": "~1.0.8",
-    "qs": "2.4.2",
+    "qs": "3.1.0",
```

Comment:
Well, the PR is a mix; I see you bumped our version by one minor, but you are pulling in deps that expose new features in our API, which would require a minor version bump.

I also have not looked at exactly what the path-to-regexp changes are, which are very important to know. I think someone already has a PR open for that dep and the upgrade actually makes us change some of our code, which I don't see in this PR.